pkgs/development/ocaml-modules/sel/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchurl, ppxlib, ppx_deriving, buildDunePackage }:
+
+buildDunePackage rec {
+  pname = "sel";
+  version = "0.4.0";
+
+  src = fetchurl {
+    url = "https://github.com/gares/sel/releases/download/v${version}/sel-${version}.tbz";
+    hash = "sha256-Sp0eSeKyFmMvOQrebqLKP+HbHQB0D0eAmEjDzxnFL1I=";
+  };
+
+  propagatedBuildInputs = [
+    ppxlib
+    ppx_deriving
+  ];
+
+  meta = {
+    description = "A simple event library";
+    homepage = "https://github.com/gares/sel/";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.maximedenes ];
```

Comment:
You may also leave the `sel.meta.maintainers`  field empty or unset. You can also put myself there, if that’s convenient.

Reply:
Thanks. I think it can make sense to put your name, as sel will also be used for the Jasmin and Easycrypt language servers. It is actually the main reason to package it separately on nixpkgs.

Reply:
Fixed.